README.md

```diff
@@ -613,51 +615,97 @@ There are many options for training the model with your synthetic data-enhanced
 
 > **Note:** **Every** `ilab` command needs to run from within your Python virtual environment.
 
-#### Train the model locally on Linux
+#### ilab model training pipelines
+
+`ilab model train` has three pipelines: `simple`, `full`, and `accelerated`.
```

Comment:
What is the default value of the `--pipeline` option when not setting the option? Or is the `--pipeline` option always required? I want to see the topic is explained in a document.

Reply:
pipeline's default is `accelerated` I will add this!